CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
I'm french but use QWERTY US FTW. :trollface: 

> I searched the docs in this repo. We use two ticks ~1700 times, one tick ~400 times...
 Add a line note

The most used method is not necessary the best. :wink: